server/routes/nl/api.py

```diff
@@ -193,3 +191,15 @@ def history():
       not current_app.config['NL_MODEL']):
     flask.abort(404)
   return json.dumps(bt.read_success_rows())
+
+
+@bp.route('/feedback', methods=['POST'])
+def feedback():
+  if (os.environ.get('FLASK_ENV') == 'production' or
+      not current_app.config['LOG_QUERY']):
+    flask.abort(404)
+
+  session_info = request.json('session_info')
+  data = request.json['data']
```

Comment:
maybe call it `feedback_data` or something?

Currently `session_info` has `id` and a list of `items` one per query:
https://github.com/datacommonsorg/website/blob/4ea4f28a7f822b6becf0e258d4799cf791e944b2/server/lib/nl/fulfillment/context.py#L138-L155

Reply:
Done